### DISCUSSION TITLE
Ability to compare user defined extensions

### DISCUSSION THREAD

**Discussion by maximoto1** [hsoft/dupeguru#1093] [Ideas]
Currently the only options one can choose are standard, music, and picture. Could you provide a "custom" field that allows the user to enter any file extension for the comparison? That way we could enter something like "mp4" or "zip", and dupeguru would compare only those files.

I realize that we can apply a post-scan filter, but I this would speeding up the initial file comparison a lot!

GREAT PROGRAM! Thank you all for your contribution to make it better with every version!

**Comment by Anton-V-K** [comment]
Nice idea!
I'd also add the ability to exclude files by extension.

**Comment by myokeeh** [comment]
Much needed.
